content/en/docs/guides/interceptors.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Interceptors
+description: >-
+  Explains how interceptors can be used for implementing generic behavior that
+  applies to many RPC methods.
+---
+
+### Overview
+
+The core of making gRPC services is implementing RPC methods. But some
+functionality is independent of the method being run and should apply to all or
+most RPCs. Interceptors are well suited to this task.
+
+### When to Use Interceptors
+
+You may already be familiar with the concept of interceptors, but may be used to
+calling them "filters" or "middleware." Interceptors are very well suited to
+implementing logic that is not specific to a single RPC method. They are also
+easy to share across different clients or servers. Interceptors are an important
+and frequently-used way to extend gRPC. You might find some functionality you
+want is already available as an interceptor in the wider gRPC ecosystem.
```

Comment:
Do we have any examples we could link out to?

Reply:
I don't want to get into curating a list or choosing winners, at least not within the scope of this document.